gui/src/lib/api.ts

```diff
@@ -359,7 +363,11 @@ export const api = (() => {
         return networks;
       },
 
-      async apply() {
+      async destroySelf(): Promise<void> {
```

Comment:
Why not call this "destroy" to match the underlying method? Ideally, this api.ts file will be code-gen'd in the not-too-distant future. I've been occasionally working on http://github.com/deref/josh-js for that.

Reply:
Done